sdk/component/conf.go

```diff
@@ -19,10 +19,15 @@ const (
 	errReasonCannotBeNil      = "cannot be nil"
 
 	// Env vars.
-	envVarKeyComponentName   = "SMITHY_COMPONENT_NAME"
-	envVarKeyWorkflowID      = "SMITHY_WORKFLOW_ID"
+	// -- BASE
+	envVarKeyComponentName = "SMITHY_COMPONENT_NAME"
+	envVarKeyWorkflowID    = "SMITHY_WORKFLOW_ID"
+	// -- LOGGING
 	envVarKeyLoggingLogLevel = "SMITHY_LOG_LEVEL"
-	envVarKeyBackedStoreType = "SMITHY_BACKEND_STORE_TYPE"
+	// -- STORE
+	envVarKeyBackendStoreType        = "SMITHY_BACKEND_STORE_TYPE"
```

Comment:
these environment variables could be simplified into one environment variable with a URI like `sqlite:///home/user/bla/smithy.db`